domains/certificates/Query.API/API/DataSyncSyncer/Service/DataSync.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using CertificateEvents.Primitives;
+using IdentityModel.Client;
+using Microsoft.Extensions.Logging;
+using Microsoft.IdentityModel.Tokens;
+
+namespace API.DataSyncSyncer.Service.Datasync;
+
+public class DataSync : IDataSync
+{
+    private readonly HttpClient httpClient;
+    private readonly ILogger<DataSync> logger;
+
+    public DataSync(HttpClient httpClient, ILogger<DataSync> logger)
+    {
+        this.logger = logger;
+        this.httpClient = httpClient;
+    }
+
+    public async Task<List<DataSyncDto>> GetMeasurement(string gsrn, Period period, string meteringPointOwner)
+    {
+        logger.LogInformation(
+            "Fetching data in period from {from} to: {to}", period.DateFrom, period.DateTo
```

Comment:
Suggestion: This will print epoch time, so I suggest to write it in a more friendly format